java/ingest/ingest-status-store/src/main/java/sleeper/ingest/task/status/DynamoDBIngestTaskStatusFormat.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.ingest.task.status;
```

Comment:
This is the same package name used in the ingest-core module. Maybe something like sleeper.ingest.status.store.task?

There's a similar problem in compaction-status-store as well.